feedback.md

```diff
@@ -0,0 +1,25 @@
+# info343-final-project
+
+Our team is proposing the creation of a React-based trivia game web app. We will use one of the available trivia question APIs to create a web-based game with the following functionality:
+
+- User accounts, managed by Google Firebase
+- Sign-in/Sign-out
+- API manipulation, to pull down questions, using the Open Trivia DB (opentdb.com)
+- A timer (10 seconds to answer each question, for example)
+- Leaderboard rankings
+- New questions every day
+- Bootstrap look and feel
+
+New visitors to the site will be asked to create a new account or sign in to an existing one. Then, they will be asked to begin that day’s Trivia Challenge, 10 questions pulled randomly from the API (though the same 10 questions for everyone who visits that day.) They will be given five minutes to answer the 10 questions which will come from a variety of subjects and will include true-false and multiple-choice questions. At the end of the challenge, users will receive a score and a standing among the day’s leaderboard of other users who have taken the challenge.
+
+Users will not be allowed to take the challenge more than once a day, but they will be able to view the current leaderboard at any time.
+
+### Responsibilities will be divided as follows:
+
+Will Tucker – User account management and sign-in/sign out functionality
```

Comment:
Seems like a good starter task, but I have to assume this will include deeper development on user profiles, photos, badges, etc.